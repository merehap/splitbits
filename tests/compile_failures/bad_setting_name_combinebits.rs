use splitbits::*;

fn main() {
    let a: u8 = 0b1010_0101;
    combinebits!(explode=panic, "aaaa aaaa");
}
