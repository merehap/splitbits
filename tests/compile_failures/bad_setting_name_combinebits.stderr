error: proc macro panicked
 --> tests/compile_failures/bad_setting_name_combinebits.rs:5:5
  |
5 |     combinebits!(explode=panic, "aaaa aaaa");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: assertion `left == right` failed: Only the 'overflow' setting is supported, but found 'explode'.
            left: "explode"
           right: "overflow"
